src/UserAccountDataChecker.php

```diff
@@ -24,13 +24,13 @@ class UserAccountDataChecker extends DataChecker {
 	public $mUser = null;
 
 	public function __construct() {
-		$this->addCheck( array( &$this, 'checkDomainValidity' ), array() );
-		$this->addCheck( array( &$this, 'checkDomainUser' ), array() );
-		$this->addCheck( array( &$this, 'checkCreatePermissions' ), array() );
-		$this->addCheck( array( &$this, 'checkSorbs' ), array() );
-		$this->addCheck( array( &$this, 'checkUserExists' ), array() );
-		$this->addCheck( array( &$this, 'checkPasswordLength' ), array() );
-		$this->addCheck( array( &$this, 'checkEmailValidity' ), array() );
+		$this->addCheck( array( $this, 'checkDomainValidity' ), array() );
```

Comment:
> Oh I thought this that was a joke. In a sense this actually looks like a joke to non 

This is the reason why I got the confirmation form PHP-core in https://bugs.php.net/bug.php?id=73751.